Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -687,9 +687,9 @@ function vtkRenderWindowInteractor(publicAPI, model) {
     // Here the first spin value is "recorded", and used to normalize
     // all the following mouse wheel events.
     if (model.wheelTimeoutID === 0) {
-      // 0.4 is roughly half-way between a large trackpad first event and small
+      // >=0.3 will permit us to detect mouse wheel events from certain mice with low spinY values
```

Comment:
this line does not make sense with next line.

It is better to use the passive form (will permit us to detect -> allows to detect).

I would keep the information that the threshold is meant to split between "mice" and "trackpads"

Reply:
Gotcha. I've update the comment 👍